src/backend/cc.d

```diff
@@ -1622,43 +1624,35 @@ extern __gshared Declar gdeclar;
  *      DTcoff          offset into code segment
  */
 
-//struct dt_t
```

Comment:
I suggest in the future to use block comments where possible to avoid unhelpful diffs like this.